using-timescaledb/continuous-aggregates.md

```diff
@@ -181,9 +181,7 @@ SELECT drop_chunks(INTERVAL '30 days', 'device_readings')
 You can read more about data retention with continuous aggregates in
 the [*Data retention*][sec-data-retention] section.
 
->:TIP: Most times the continuous aggregate view will be updated by the background job;
-  however, if you would like to run it yourself, you may use the
-  [`REFRESH MATERIALIZED VIEW` command][api-refresh-continuous-aggs].
+>:TIP: Most times the continuous aggregate view will be updated by the background job; however, if you would like verify that the view is working correctly, you may use the [`REFRESH MATERIALIZED VIEW` command][api-refresh-continuous-aggs].
```

Comment:
I don't understand how calling the refresh will verify that cagg bgw works correctly. Thus I feel the current formulation is misleading.

Reply:
Because running `REFRESH MATERIALIZED VIEW` will dispatch a job to update the view, so it will indirectly check that the view works. Several users have stumbled over that they set up the view and then don't see it populated, so we have to direct them to use `REFRESH MATERIALIZED VIEW`.